### ISSUE TITLE
Feature Request: Freeze Columns in Data View.

### ISSUE THREAD

**Issue by eric-schleicher** [structr/structr#334]
Seems so silly, but it's a pain to scroll in the data view and be sure that the columns that you want to edit is the right one (especially if hey are booleans).  I generally set record limits that are longer than 10 records...  keeping columns straight is made more difficult given that they aren't always displayed in the same order.

Having columns names/headers stay put and a data grid that scrolls would be a nice usability improvement

**Comment by whatSocks** [comment]
+1

**Comment by vigorouscoding** [comment]
I investigated this earlier and tried my luck once again. I came up with a solution which *only* works in WebKit-based browsers (tested in Mac Safari / Mac Chrome). If you are using such a browser you will hopefully be happy with the solution.
It makes use of the experimental `position: sticky;` CSS feature - so we don't know when/if other browsers will adopt it. Let me know what you think.

(I gave it multiple tries with other JS/CSS approaches but none of them provided solutions for all of the requirements - this is hopefully some kind of middle ground)
